### PR TITLE
Insights Snapshot badge

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Similar to status badges, the Insights Snapshot badge provides a quick view of insights metrics for a project. This new doc describes how to create a badge (using a tool that generates an embeddable code snippet) for both public and private repositories, via Project Settings. 

This doc includes only one screenshot of a sample badge, and does not include screenshots of the Settings page within the web app. This is to make it easier to keep the doc updated/maintained if there are changes in the UI (as this feature is new and could still undergo changes soon based on feedback from users). I did include an example of a badge code snippet, generated for the CircleCI docs repo. 

If it's better to include UI screenshots at this point, and/or not include the code snippet, we can definitely change that here.

### CODE REVIEW

File: jekyll/_cci2/insights-snapshot-badge.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+:toc: macro
+:toc-title:
+
+== Overview
+
+The Insights Snapshot badge, similar to a <<status-badges#,status badge>> that is commonly embedded in project READMEs, provides a quick view of your project's health and usage metrics.   
+
+image::insights-snapshot-preview.png[Insights snapshot badge]
+
+The most common use case for the Insights Snapshot badge is to display <<insights#,insights>> metrics for the deployment workflow on project’s main branch in the README. With that said, the badge can be configured to display insights metrics for any workflow on any branch within a project. This enables you to create additional badges for your scheduled workflows, workflows with different build targets, or workflows specifically used for testing, to name a few examples.
```
Comment (catherinepope) [inline jekyll/_cci2/insights-snapshot-badge.adoc:19]:
```suggestion
The most common use case for the Insights Snapshot badge is to display <<insights#,insights>> metrics for the deployment workflow on the project’s main branch in the README. With that said, the badge can be configured to display insights metrics for any workflow on any branch within a project. This enables you to create additional badges for your scheduled workflows, workflows with different build targets, or workflows specifically used for testing, to name a few examples.
```

File: jekyll/_cci2/insights-snapshot-badge.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+To generate an Insights Status badge code snippet for a public repository, go to the https://app.circleci.com/[CircleCI web app], and select a specific project. Then, open *Project Settings* and navigate to the *Insights Snapshot Badge* page. 
+
+On the Insights Snapshot Badge page, you can select the branch, workflow, and time window for which you want to display project metrics for, as well as the desired format for the code snippet.
+
+You can generate code in the following formats:
+
+* Image URL
+* Markdown
+* Textile
+* Rdoc
```
Comment (catherinepope) [inline jekyll/_cci2/insights-snapshot-badge.adoc:39]:
```suggestion
* RDoc
```

**Review by catherinepope — APPROVED** [review]
👍  Looks really good! Just made a couple of tiny amendments.

**Comment by devinmarieb** [comment]
@kbatuigas Just wondering if we want to remove capitalization of "Insights"/"Insights Badges" etc.

File: jekyll/_cci2/insights-snapshot-badge.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+== Overview
+
+The insights snapshot badge, similar to a <<status-badges#,status badge>> that is commonly embedded in project READMEs, provides a quick view of your project's health and usage metrics.   
+
+image::insights-snapshot-preview.png[Insights snapshot badge]
+
+The most common use case for the insights snapshot badge is to display <<insights#,insights>> metrics for the deployment workflow on the project’s main branch in the README. With that said, the badge can be configured to display insights metrics for any workflow on any branch within a project. This enables you to create additional badges for your scheduled workflows, workflows with different build targets, or workflows specifically used for testing, to name a few examples.
+
+While commonly placed in a project README, insights Snapshot badgegs can be embedded in any web document. The badge currently displays the following three metrics from the insights dashboard:
```
Comment (anthony-j-castro) [inline jekyll/_cci2/insights-snapshot-badge.adoc:21]:
```suggestion
While commonly placed in a project README, insights snapshot badges can be embedded in any web document. The badge currently displays the following three metrics from the insights dashboard:
```

File: jekyll/_cci2/insights-snapshot-badge.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+* *Throughput* - The average number of runs of a workflow over the selected time window.
```
Comment (anthony-j-castro) [inline jekyll/_cci2/insights-snapshot-badge.adoc:23]:
```suggestion
* *Throughput* - The average number of runs of a workflow, over the selected time window.
```

To be consistent with the other bullets. 🙃

**Review by anthony-j-castro — APPROVED** [review]
LGTM!

File: jekyll/_cci2/insights-snapshot-badge.adoc (hunk 3 above)
Comment (tk-cc-1) [inline jekyll/_cci2/insights-snapshot-badge.adoc:21]: typo: Insights Snapshot badges

File: jekyll/_cci2/insights-snapshot-badge.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* *Duration* - The 95th percentile duration of the selected workflow, over the specified time window.
+    95th percentile gives you an idea of the longest duration or time you’re waiting for a build to finish, excluding the last 5 percent, to account for potential transitory spikes.
+* *Success Rate* - The percentage of successful builds for the selected workflow, over the specified time window.
+
+== Generating an insights snapshot badge
```
Comment (tk-cc-1) [inline jekyll/_cci2/insights-snapshot-badge.adoc:28]: noticed we have diff capitalization throughput page

**Pull request merged**